README.md

```diff
@@ -2,18 +2,29 @@
+If you're interested in contributing to this repository, please clone the repo, make changes, and open a PR ✨. Please review the guidelines in the [CONTRIBUTING file](.CONTRIBUTING.md) before making any changes.
 
-Thank you for taking your time to contribute to this repository 💜.
+Thank you for taking the time to contribute to this repository 💜.
 
 ## Disclaimer
 
-This developer documentation has been created to support and promote the Polkadot ecosystem. The information presented herein is made available solely for general information purposes. The Organizations do not warrant the accuracy, completeness, or usefulness of this information. Any reliance you place on such information is strictly at your own risk. The Organizations disclaim all liability and responsibility arising from any reliance placed on this information by you or by anyone who may be informed of any of its contents. All statements and/or opinions expressed in these materials are solely the responsibility of the person or entity providing those materials and do not necessarily represent the opinion of the Organizations. The information should not be construed as professional or financial advice of any kind. Advice from a suitably qualified professional should always be sought in relation to any particular matter or circumstance. The information herein may link to or integrate with other websites operated or content provided by third parties, and such other websites may link to this website. The Organizations have no control over any such other websites or their content and will have no liability arising out of or related to such websites or their content. The existence of any such link does not constitute an endorsement of such websites, the content of the websites, or the operators of the websites. These links are being provided to you only as a convenience, and you release and hold the Organizations harmless from any and all liability arising from your use of this information or the information provided by any third-party website or service."
+### Licensing Disclaimer
+
+The developer documentation provided herein has been collaboratively created and curated by [Web 3.0 Technologies Foundation](https://web3.foundation/), [Parity Technologies Ltd.](https://www.parity.io/), and [PaperMoon Dev SL](https://papermoon.io) (hereinafter referred to as the "Organizations") as a joint effort to support the Polkadot ecosystem.
+
+### Documentation License
+The content of these documents is [licensed](LICENSE.md) under the Creative Commons Attribution 4.0 International License ([CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)). You are free to share, copy, redistribute, and adapt the material for any purpose, even commercially, provided that appropriate credit is given to the Organizations, a link to the license is provided, and any changes made are indicated.
```

Comment:
are we using the {target=\_blank} modifier for these links as well? 
Like:
```suggestion
The content of these documents is [licensed](LICENSE.md) under the Creative Commons Attribution 4.0 International License ([CC BY 4.0](https://creativecommons.org/licenses/by/4.0/){target=\_blank}). You are free to share, copy, redistribute, and adapt the material for any purpose, even commercially, provided that appropriate credit is given to the Organizations, a link to the license is provided, and any changes made are indicated.
```

same for the rest of the links here

Reply:
Unfortunately, GitHub does not support the target attribute. Even note that we have it for mkdocs but it does nothing